acoustic_msgs/msg/MultibeamWatercolumn.msg

```diff
@@ -0,0 +1,60 @@
+
+# The header time should be the time the ping was transmitted, not
+# the time the message was received.
+# Note that these two could potentially be several seconds apart,
+# especially in deep water.
+std_msgs/Header header
+
+float32 sound_speed   # m/s
```

Comment:
Add comment that this is sound speed typically measured at the head, not to be confused with an average watercolumn sound speed for example.